api/docs/docs/updates.md

```diff
@@ -7,6 +7,15 @@ description: Updates to the Covid Act Now API.
 
 Updates to the API will be reflected here.
 
+### Vaccine Booster data now available
+_Added on 2022-01-13_
+
+Vaccine booster shot (or additional dose) data is now available within the Covid Act Now API.
+
+Fields added:
+ * `vaccinationsAdditionalDose`: Number of individuals who are fully vaccinated and have received a booster (or additional) dose.
+* `vaccinationsAdditionalDoseRatio`: Ratio of population that are fully vaccinated and have received a booster (additional) dose.
```

Comment:
nits:
1.  leading whitespace is different between these two bullets.
2. should the second line be "or additional" like the first one?
3. Should we include the actuals/metrics prefixes?  i.e. `actuals.vaccinationsAdditionalDose` and `metrics.vaccinationsAdditionalDoseRatio` ?